flink-table/flink-sql-parser/src/test/java/org/apache/flink/sql/parser/FlinkSqlParserImplTest.java

```diff
@@ -286,6 +286,12 @@ void testShowColumns() {
                 .ok("SHOW COLUMNS IN `CATALOG1`.`DB1`.`TBL` NOT LIKE '%'");
     }
 
+    @Test
+    public void testShowPartitions() {
+        sql("show partitions tbl").ok("SHOW PARTITIONS `TBL`");
```

Comment:
add a test like
`show partitions c1.d1.tbl`
The [SqlToOperationConverterTest.java](https://github.com/apache/flink/pull/19553/files#diff-399da8a41d837c40407dfafce96c2cd186386e5f7931422b2f87b9dafdeb9dfd)#testShowPartitions is same.